### ISSUE TITLE
Save / Load Fx Chains

### ISSUE THREAD

**Issue by sense-amr** [surge-synthesizer/surge#5476]
The ability to save FX Chain setups and Load them on demand would be awesome..
I do note the mention of FX Chains in the current XT Surge version .. with the option to Clear them ..

**Comment by baconpaul** [comment]
Yeah we would love to do this too. After we ship xt1 ships though.

**Comment by sense-amr** [comment]
sure its  a feature request.. for sure a feature for the future ..

**Comment by baconpaul** [comment]
yeah it's a good idea and one we've discussed. i don't think its in an issue either, and now it is :)

**Comment by sense-amr** [comment]
i might also request a distinctive colour atleast for [A] and [B] FX areas.. that way its a bit easier to see what you're working on .. espcially if you have CH on both and accidentally change the WRONG one .. on SCENE A when you thought you  are modifying the CH on scene B

**Comment by sense-amr** [comment]
the more things a user has to remember and think about when making a patch like this the less likely they are to have a good and free experience just making one .. 
because they will make a mistake modify the wrong thing and ruin their patch .. then give up

**Comment by sense-amr** [comment]
and if anyone says the A and B in FX are super clear and distinctive from each other in the current UI .. im going have to strongly disagree

**Comment by mkruselj** [comment]
Feel free to edit the skin SVGs to your taste. That's possible today. At any rate it seems people are not having trouble differentiating A and B since we got no such reports in over 2 years.

**Comment by baconpaul** [comment]
In a future skin engine, we will also have panels-per-effect and we can make those panels aware of the slot they are in pretty easily, so you could do even more ambitious stuff with the skin engine.
